TLM/TLM/UI/LegacySubTool.cs

```diff
@@ -24,29 +24,32 @@ public abstract class LegacySubTool {
         private Texture2D windowTexture_;
 
         protected GUIStyle WindowStyle =>
-            windowStyle_ ?? (windowStyle_ = new GUIStyle {
+            // ReSharper disable once ConvertToNullCoalescingCompoundAssignment
+            windowStyle_ ??
+            (windowStyle_
+                 = new GUIStyle {
                                     normal = {
```

Comment:
wow! what is happening to the indent spaces in here? if you like it would be nice to reduce the indent.

Reply:
Rider IDE was suggesting to use new language feature, so i had to add a suppress comment. Once the feature is in the supported features, we can apply that suggestion. Also formatted it on several lines otherwise it shoots off screen.